### ISSUE TITLE
Got an error during the TestSVM, no img_data.json file

### ISSUE THREAD

**Issue by minzhou1003** [sausax/pose_estimation#2]
I followed your instruction and I did Preprocessing and SVM train, but I got an error during the TestSVM:
```
$./TestSVM
libc++abi.dylib: terminating with uncaught exception of type nlohmann::detail::parse_error: [json.exception.parse_error.101] parse error at 1: syntax error - unexpected end of input; expected '[', '{', or a literal
Abort trap: 6
```
Then after debugging ```TestSVM.cpp```, I figure out this is because in the main func the``` img_data.json``` doesn't exist under jsons after preprocessing and training. As a result, the ```file_contents="" ```so, the argument```""```is passed to the ```json::parse ```raising the error.
I don’t know where am I wrong but I think I've run the training part correctly,  please help me, thanks!

**Comment by sausax** [comment]
Are you getting any error during preprocessing ?

**Comment by minzhou1003** [comment]
Yes, just like above description, but I eventually fixed it by modifying the preprocessing part, now it works well and can generate the ```img_data.json```. 

Now, I have one question about the preprocessing part, so I want to estimate all joints that visible in one image, but your code is only for right shoulder, right elbow and right wrist, right? Can you give me any suggestions about this? Thanks a lot for your response!

**Comment by sausax** [comment]
I only used 3 points for simplification. Ideally all 10 joints should be used.
